tools/checkpoint_conversion/convert_resnet_vd_checkpoints.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/env python3
+
```

Comment:
can you add instructions to run this file on the top of the file?

Reply:
Done.